### PR TITLE
Re-arrange configuration chapter

### PR DESCRIPTION
**Pull request by mwithi**
The idea is to provide ONE method as preferred and an alternative, rather than TWO methods that the user has to choose ("use this or this") without any idea of the difference.

@dbmalkovsky @mizzioisf

### CODE REVIEW

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -390,22 +390,53 @@ NOTE: Remember to set the updated database password also in the Open Hospital co
...
 
 * the *IP address* of the the MariaDB/MySQL database server hosting the the Open Hospital database. Replace “localhost” with the server IP address (e.g., 192.168.1.100)
 * the *TCP Port Number* of the database server - default is “3306"
 * the *Database name* of the OH installation - default is "oh"
 * the *Database user* of the OH database - default is "isf"
 * the *Database password* associated with the user
 
+Please set them according to the values specified in the installation phase:
+
+in **oh.sh / oh.ps1** scripts:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: These seems strange to me; line 403 ends with a colon and so does 405; are the two line suppose to be connected?

File: doc_admin/AdminManual.adoc [hunk 2]
```diff
@@ -390,22 +390,53 @@ NOTE: Remember to set the updated database password also in the Open Hospital co
...
+Please set them according to the values specified in the installation phase:
+
+in _database.properties_ config file:
+```
+jdbc.url=jdbc:mysql://127.0.0.1:3306/oh
+jdbc.username=isf
+jdbc.password=isf123
+```
+
+in _log4j.properties_ config file>
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: why does this line end in a `>`?

File: doc_admin/AdminManual.adoc [hunk 3]
```diff
@@ -390,22 +390,53 @@ NOTE: Remember to set the updated database password also in the Open Hospital co
...
+$script:DATABASE_PASSWORD="isf123"
+```
+
+After you can generate the config files: on Linux execute *oh.sh -g*, on Windows launch *oh.bat* and choose the option *g*.
+
+*Alternative method*: you can first generate the config files with the *-g* and after directly modify the _database.properties_ and _log4j.properties_ configuration files, located in the _oh/rsc/_ resource folder, without generating them every time.
+
+Please set them according to the values specified in the installation phase:
+
+in _database.properties_ config file:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: as there are two files to examine; would it make more sense to make a ordered or unordered list to highlight the two files?     Again there is the issue with two adjacent lines ending in a colon.

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: let me remove the second

File: doc_admin/AdminManual.adoc (hunk 2 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: it was supposed to be a colon, but as for the other paragraph I remove it

File: doc_admin/AdminManual.adoc (hunk 3 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: removed one

**Pull request merged**